talisker/gunicorn.py

```diff
@@ -179,6 +194,24 @@ def init(self, parser, opts, args):
 
         return cfg
 
+    def load_config(self):
+        super(TaliskerApplication, self).load_config()
+
```

Comment:
at this point, self.cfg is the full config, including app defaults, config file, and cli args.

So, I think we should do the worker_exit hook here, rather than init. a) it allows us to inspect workers b) we can detect if the user has specified a custom worker_exit, and wrap it?